adam/flask_backend/app.py

```diff
@@ -0,0 +1,108 @@
+from glob import glob
+from http import HTTPStatus
+from typing import Any
+
+import yaml
+from flask import Flask, abort, request
+from flask_cors import CORS
+
+from adam.paths import (
+    DATA_DIR,
+    LEARNERS_DIR,
+    SITUATION_DIR_NAME,
+    POST_LEARN_FILE_NAME,
+    EXPERIMENTS_DIR_NAME,
+    EXPERIMENTS_TESTING_DIR_NAME,
+)
+from adam.experiment.experiment_data_loader import (
+    get_learners,
+    get_train_curricula,
+    get_test_curricula,
+)
+
+app = Flask(__name__)
+cors = CORS(app)
+app.config["CORS_HEADERS"] = "Content-Type"
+app.config["JSON_SORT_KEYS"] = False
+
+
+@app.route("/api/learners", methods=["GET"])
+def get_all_learners() -> Any:
+    """Get all learner configurations which are prepared."""
+    learners = get_learners(DATA_DIR)
+    return {"learner_types": learners}
+
+
+@app.route("/api/training_curriculum", methods=["GET"])
+def get_all_train_curriculum() -> Any:
+    """Get all possible training curriculum."""
+    train_cur = get_train_curricula(DATA_DIR)
+    return {"training_curriculum": train_cur}
+
+
+@app.route("/api/training_curriculum", methods=["GET"])
```

Comment:
Is this a duplicate route? There already seems to be a route for training_curriculum, should this be testing_curriculum?